### PR TITLE
Add script to create prefab package and maven artifact

### PR DESCRIPTION
**Pull request by dturner**
Packaging and changing all the prefab config files was taking a long time so I've created a script to do it. Should save quite a bit of time for future releases.

### CODE REVIEW

File: prefab_build.sh [hunk 1]
```diff
@@ -0,0 +1,88 @@
+# Test that the prefab binary exists
```
Comment (philburk) [inline prefab_build.sh:1]: Add at top
#!/bin/bash

File: prefab_build.sh [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+if hash prefab 2>/dev/null; then
+  echo "Prefab is installed"
+else
+  echo "Prefab binary not found. See https://github.com/google/prefab for install instructions"
+  exit 1
+fi
+
+# Get the version string from the source
+major=`grep "#define OBOE_VERSION_MAJOR" include/oboe/Version.h | cut -d' ' -f3`
```
Comment (philburk) [inline prefab_build.sh:10]: Nice!
BTW, Mikhail says the back tick is obsolete and recommends:

    major=$(grep "#define OBOE_VERSION_MAJOR" include/oboe/Version.h | cut -d' ' -f3)

File: prefab_build.sh [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    echo "Copying the ${abi} library"
+    cp -v "../${abi}/liboboe.so" "oboe-${version}/prefab/modules/oboe/libs/android.${abi}/"
+  done
+
+  # Verify the prefab packages
+  for abi in ${ABIS[@]}
+  do
+
+    prefab --build-system cmake --platform android --os-version 29 \
+    --stl c++_shared --ndk-version 21 --abi ${abi} \
```
Comment (philburk) [inline prefab_build.sh:48]: indent

File: prefab_build.sh (hunk 2 above)
Comment (dturner) [reply prefab_build.sh:10]: Good to know - thanks!

File: prefab_build.sh (hunk 3 above)
Comment (dturner) [reply prefab_build.sh:48]: Not sure what you mean? It is indented already using 2 spaces.

Comment (philburk) [reply prefab_build.sh:50]: In android style, continuation lines are usually indented by 2 levels. That would be an extra 4 spaces in this case relative to line 47.

Comment (dturner) [reply prefab_build.sh]: I getcha. Thanks for the explanation - I wasn't aware of the rule. Fixed.

**Pull request merged**